test/models/friendship_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
```

Comment:
Please also include here test for association with the User model.

You can actually use the shoulda-matchers gem to make this association test easier. https://guides.railsgirls.com/testing-shoulda-matchers